Include functionalities into signup module.

jparrax opened the pull request:
New validations were included into js/validations to do client-side validations
File signupValidation was created to do the server-side validation of user name and email.
Signup view file was changed to include the new functionalities.
I deleted terms from Footer because at this moment it is not necessary.

jparrax committed:
This is a test commit

jparrax committed:
Revert "This is a test commit"

This reverts commit e6172b1648c6510f53a81bba11ea6139efb50510.

jparrax committed:
Include functionalities into signup module.

New validations were included into js/validations to do client-side validations
File signupValidation was create to do the server side validation of user name and email.
Signup view file was changed to include the new functionalities.
I deleted terms from Footer because in this moment it is not necessary.

AntonRozhanskii commented on signupValidation.php:
```diff
@@ -0,0 +1,48 @@
+    // Result in case to find something
+    if ($result->num_rows > 0) {
+        echo "This user name is being used";
+    } 
```
Rewrite this code in this way:
https://www.w3schools.com/php/php_mysql_prepared_statements.asp

AntonRozhanskii commented on signupValidation.php:
```diff
@@ -0,0 +1,48 @@
+    // Result in case to find something
+    if ($result->num_rows > 0) {
+        echo "This email is being used";
+    } 
```
Rewrite this code in this way:
https://www.w3schools.com/php/php_mysql_prepared_statements.asp

AntonRozhanskii reviewed (changes requested):
Rewrite sql query code

AntonRozhanskii commented on js/validations/signup.js:
```diff
@@ -28,13 +49,26 @@ function checkPasswordLength() {
+function checkConfirmPassword(){
+    var password = document.getElementById('password');
+    var confirmPassword = document.getElementById('confirm-password'); 
+    var confirmPassWarning = document.getElementById('confirmPassWarning');
```
First, we already use let.
Second, here is difference between them https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var

AntonRozhanskii reviewed (changes requested):
Fix

jparrax commented:
I will start to do it now, I will include the missing parts for the module. Enjoy, the world is crashing!

jparrax committed:
Include functionalities into signup module.

New validations were included into js/validations to do client-side validations
File signupValidation was created to do the server-side validation of user name and email.
Signup view file was changed to include the new functionalities.
I deleted terms from Footer because at this moment it is not necessary.
